docs/tutorial/flows.md

```diff
@@ -0,0 +1,148 @@
+---
+description: Learn the basics of defining and running flows.
+tags:
+    - tutorial
+    - getting started
+    - basics
+    - flows
+    - logging
+    - parameters
+    - retries
+---
+## What is a Flow?
+
+[Flows](/concepts/flows/) are like functions. They can take inputs, perform work, and return an output. In fact, you can turn any function into a Prefect flow by adding the `@flow` decorator. When a function becomes a flow, its behavior changes, giving it the following advantages:
+
+- State transitions are recorded, allowing for flow execution to be observed.
```

Comment:
```suggestion
- It has a [state](/concepts/states/), which determines when it can run. Transitions between states are recorded, allowing for flow execution to be observed.
```